java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -2356,8 +2356,9 @@ public CompletableFuture<String> xgroupSetId(
             @NonNull String key,
             @NonNull String groupName,
             @NonNull String id,
-            @NonNull String entriesReadId) {
-        String[] arguments = new String[] {key, groupName, id, "ENTRIESREAD", entriesReadId};
+            @NonNull Long entriesReadId) {
```

Comment:
If is a number, but not an ID, please rename throughout
```suggestion
            @NonNull Long entriesRead) {
```